### PR TITLE
Github OAuth token configuration for composer added

### PR DESCRIPTION
**Pull request by igormukhingmailcom**

### CODE REVIEW

**Comment by kosssi** [comment]
Thanks for the PR.

File: defaults/main.yml [hunk 1]
```diff
@@ -9,3 +9,7 @@ composer_home_owner: root
 composer_home_group: root
 composer_global_packages: {}
 proxy_env: {}
+
+# Visit https://github.com/settings/applications 
+# and generate personal access token
+composer_github_oauth: false
```
Comment (kosssi) [inline defaults/main.yml]: Can you add newline ;)

File: tasks/oauth.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Configure github oauth token
+  template:
+    src: ".composer/auth.json.j2"
+    dest: "~/.composer/auth.json"
+  when: composer_github_oauth != false
```
Comment (kosssi) [inline tasks/oauth.yml]: Can you add newline ;)

**Comment by igormukhingmailcom** [comment]
Not at all. 
PS. You write cool roles.

**Comment by igormukhingmailcom** [comment]
Can you bump version in galaxy, please?

**Comment by kosssi** [comment]
Done v1.3.0 ;)

**Comment by igormukhingmailcom** [comment]
Thank you :)

**Pull request merged**